macro/src/reflect.rs

```diff
@@ -63,11 +63,17 @@ impl JavaClass {
     fn parse_javap(&self, package_name: &JavaPath) -> Result<SpannedClassInfo, SpanError> {
         let mut command = Command::new("javap");
 
+        let classpath = match env::var("CLASSPATH") {
+            Ok(val) => val,
+            Err(e) => panic!("duchess cannot read the CLASSPATH environment variable: {e}"),
```

Comment:
Panicking if we can't find CLASSPATH sems a bit extreme, no? Can we just not pass a `-cp` argument in that case?